cypress/e2e/finance/funds/test-acquisition-unit-restrictions-for-funds.cy.js

```diff
@@ -91,6 +91,7 @@ describe('ui-finance: Funds', () => {
     cy.login(user.username, user.password, { path:TopMenu.fundPath, waiter: Funds.waitLoading });
     FinanceHelp.searchByName(defaultfund.name);
     Funds.checkZeroSearchResultsHeader();
+    cy.logout();
```

Comment:
why do you need a logout? isn't it obsolete?

Reply:
Fixed